client/my-sites/exporter/section-export.jsx

```diff
@@ -49,8 +49,9 @@ const SectionExport = ( { isJetpack, canUserExport, site, translate } ) => {
 			<Fragment>
 				<FormattedHeader
 					className="exporter__section-header"
-					headerText={ translate( 'Export your content' ) }
+					headerText={ translate( 'Export Your Content' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'exporting your content' )` *ES Score: 6*